test/unit/api/props-with-symbol.js

```diff
@@ -0,0 +1,73 @@
+import afterMutations from '../../lib/after-mutations';
+import hasSymbol from '../../lib/after-mutations';
+import createSymbol from '../../../src/util/create-symbol';
+import element from '../../lib/element';
+import fixture from '../../lib/fixture';
+import { props } from '../../../src/index';
+
+describe('api/props-with-symbol', () => {
+  if (!hasSymbol()) {
+    return;
+  }
+
+  let elem;
+  const secret = createSymbol('secret');
+  const secret2 = createSymbol('secret');
+
+  beforeEach(done => {
+    elem = new (element().skate({
+      props: {
+        [secret]: {
+          initial: 'secretKey',
+        },
+        [secret2]: {
+          initial: 'secretKey2',
```

Comment:
Do we have a test that actually check [this issue](https://github.com/skatejs/skatejs/issues/829) being solved?
I mean, adding 2+ props with symbol keys without description and trying to get their values?